src/pages/HomePage.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import FetchStats from '../store/api';
+import { GetStats } from '../store/reducer';
+
+const HomePage = () => {
```

Comment:
- [ ] According to the project requirements, there should be a filtering mechanism on your Homepage (like a search functionality, for example). Kindly add this.

![sunny-home](https://user-images.githubusercontent.com/36542986/142707431-c8e72c2b-4016-4db5-abd9-2a2ae9a97405.jpg)